id-repository/credential-service/pom.xml

```diff
@@ -166,7 +172,7 @@
 		<dependency>
 			<groupId>io.mosip.biometric.util</groupId>
 			<artifactId>biometrics-util</artifactId>
-			<version>${kernel.core.version}</version>
+			<version>1.2.1-SNAPSHOT</version>
```

Comment:
why you have hardcoded the version in the dependency

Reply:
only for develop branch we are using 1.2.1 snapshot its hardcoded